src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const employeesList = document.querySelector('ul');
+
+function convertSalary(salaryString) {
+  return salaryString.slice(1).replace(',', '.') * 1000;
+}
+
+function sortList(list) {
+  return [...list.children]
+    .sort((a, b) =>
+      convertSalary(b.dataset.salary) - convertSalary(a.dataset.salary))
+    .forEach(elem => list.append(elem));
```

Comment:
```suggestion
    .forEach(element => list.append(element));
```
It is better to use full names, especially if it's about saving only 2-3 chars